### PR TITLE
style: use grid template for dashboard pending txs

### LINKED ISSUE #914: [Mobile] Dashboard transaction queue elements cramped together
## Bug description
The elements are all cramped together. We could give them the same treatment as the tx list has in the PR #910 

## Environment
 - Browser: Chrome Mobile
 - Env: https://web-core.pages.dev/

## Steps to reproduce
 1. Go to this safe https://web-core.pages.dev/eth:0x8675B754342754A30A2AeF474D114d8460bca19b/home

## Expected result
Similar to this organization
image

## Obtained result
image

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves

Resolves #914

## How this PR fixes it
Uses grid templates to style the Transaction queue items in the Dashboard

## How to test it
1. Open a Safe with existing pending transactions
2. Go to Dashboard
3. Reduce the viewport width and observe the pending transactions do not get cramped.
4. Toggle mobile viewport observe the pending transactions do not get cramped.

## Analytics changes
N/A

## Screenshots
Screenshot 2022-10-18 at 13 18 24

### CODE REVIEW

**Review by katspaugh — APPROVED** [review]
Nice! ❤️

**Comment by usame-algan** [comment]
Can we display the values in one row as long as there is enough space for them?
Screenshot 2022-10-18 at 17 09 49

**Comment by DiogoSoaress** [comment]
> Can we display the values in one row as long as there is enough space for them? Screenshot 2022-10-18 at 17 09 49

I've made some changes in the last commit @usame-algan

File: src/components/dashboard/PendingTxs/PendingTxListItem.tsx [hunk 1]
```diff
@@ -53,36 +18,33 @@ type PendingTxType = {
...
-                </StyledConfirmationsCount>
-              ) : (
-                <Spacer />
-              )}
-              <ChevronRight color="border" />
-            </TxConfirmations>
-          </Grid>
-        </StyledContainer>
-      </a>
+      <Box className={classNames(css.gridContainer, css.columnTemplate)}>
```
Comment (usame-algan) [inline src/components/dashboard/PendingTxs/PendingTxListItem.tsx:21]: Is there a reason for removing the `<a>` element here? Now the `div` gets the href and the cursor doesn't change when hovering.

File: src/components/dashboard/PendingTxs/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.gridContainer {
+  width: 100%;
+  display:grid;
+  grid-gap: 4px;
+  align-items: center;
+  padding: 12px 16px;
```
Comment (usame-algan) [inline src/components/dashboard/PendingTxs/styles.module.css:6]: This was `8px 16px` before which made the accordion look slightly smaller. Is this change intended?

File: src/components/dashboard/PendingTxs/PendingTxListItem.tsx (hunk 1 above)
Comment (DiogoSoaress) [reply src/components/dashboard/PendingTxs/PendingTxListItem.tsx:21]: Oops good catch! It was not intended

File: src/components/dashboard/PendingTxs/styles.module.css (hunk 2 above)
Comment (DiogoSoaress) [reply src/components/dashboard/PendingTxs/styles.module.css:6]: Great eye ser! Also not intended

**Review by usame-algan — APPROVED** [review]
🚀

**Comment by francovenica** [comment]
> Can we display the values in one row as long as there is enough space for them? 

Idk how much room there is, specially with long decimals. This is an iphone 12 viewport (using the chrome emulator)
image

**Comment by francovenica** [comment]
To me it looks good, much better than before. I think at some point we will need a mobile design for this and the Transactions tabs, so we can be more picky later
